src/app/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -394,6 +394,12 @@ export class PdfViewerComponent
 
     const eventBus = createEventBus(PDFJSViewer);
 
+    eventBus.on('pagechanging', e => {
```

Comment:
Looks like this is never triggered. Did it work for you?

Reply:
yes, I tested it. 'pagechanging' used to be called 'pagechange', but got renamed. maybe that's making the difference

Reply:
Could you specify, how you tested it? Did you set "show all pages" to false? Could you provide the PDF you used? Since it's working fine for me it's difficult to fix the issue without further information.